vpr/src/route/MixedNetlistRouter.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+/** @file Mixed case for NetlistRouter */
```

Comment:
Explain more, say nested parallelism, explain what that is.

Reply:
Explain that it avoids using TBB so it can devote some threads to coarse-grained (net- or part of net) parallelism and some to fine-grained (wave expansion) paralellism.